code/modules/admin/verbs/adminhelp.dm

```diff
@@ -668,7 +668,9 @@ GLOBAL_DATUM_INIT(ahelp_tickets, /datum/admin_help_tickets, new)
 //
 
 /client/proc/giveadminhelpverb()
-	src.verbs |= /client/verb/adminhelp
+	if(!src)
```

Comment:
This shouldn't be necessary, as it's checked before calling this proc.

Reply:
Call proc, client leaves, this triggers, runtime.